websites/L/Lidraughts/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "377284540420587541",
+		"name": "unicornbetrayal#7888"
```

Discriminator added mistakenly by outdated pmd version
```suggestion
		"name": "unicornbetrayal"
```